js/lcx.js

```diff
@@ -0,0 +1,579 @@
+        const canceledAmount = this.safeFloat (order, 'cancelled_quantity');
+        if (canceledAmount !== undefined) {
+            remaining = this.sum (remaining, canceledAmount);
+        }
+        const amount = this.safeFloat (order, 'Amount');
+        const cost = this.safeFloat2 (order, 'Cost');
+        if (type === 'market') {
+            price = undefined;
+        }
+        const average = this.safeFloat (order, 'Average');
+        let clientOrderId = this.safeString (order, 'client_order_id');
+        if (clientOrderId === '') {
+            clientOrderId = undefined;
+        }
+        return {
+            'id': id,
+            'info': order,
+            'clientOrderId': clientOrderId,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'status': status,
+            'price': price,
+            'amount': amount,
+            'filled': filled,
+            'remaining': remaining,
+            'average': average,
+            'cost': cost,
+            'fee': undefined,
+            'trades': undefined,
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'Pair': market['symbol'],
+            'Amount': parseFloat (this.amountToPrecision (symbol, amount)),
+            'Price': parseFloat (this.priceToPrecision (symbol, price)),
+            'OrderType': type.toUpperCase (),
+            'Side': side.toUpperCase (),
+        };
+        const response = await this.privatePostCreate (this.extend (request, params));
+        const data = this.safeValue (response, 'data');
+        return this.parseOrder (data);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        const request = {
+            'OrderId': id,
+        };
+        const response = await this.privatePostCancel (this.extend (request, params));
+        const data = this.safeValue (response, 'data');
+        return this.parseOrder (data);
+    }
+
+    nonce () {
+        return this.milliseconds ();
+    }
+
+    sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
+        let url = this.urls['api'][api] + '/';
+        const query = this.omit (params, this.extractParams (path));
+        if (api === 'public') {
+            if (method === 'POST') {
+                url += this.implodeParams (path, params);
+                body = this.json (query);
+            } else {
+                url += this.implodeParams (path, params);
+                if (Object.keys (query).length) {
+                    url += '?' + this.urlencode (query);
+                }
+            }
+        } else if (api === 'private') {
```

Comment:
Unnecessary branching
```suggestion
        }  

if (api === 'private') {
```